api/config/config.go

```diff
@@ -99,8 +100,10 @@ type BaseImageConfig struct {
 	DockerfilePath string `validate:"required" json:"dockerfilePath"`
 	// GCS URL Containing build context
 	BuildContextURI string `validate:"required" json:"buildContextURI"`
+	// Path to sub folder which is intended to build instead of using root folder
+	BuildContextSubPath string `json:"buildContextSubPath"`
 	// path to main file to run application
-	MainAppPath string `validate:"required" json:"mainAppPath"`
+	MainAppPath string `json:"mainAppPath"`
```

Comment:
Why has the `validate:"required"` part been removed now? We're not somehow setting the default value elsewhere, so my understanding is we still need it. 🤔